numba/tests/test_unicode.py

```diff
@@ -1086,6 +1094,43 @@ def pyfunc(x):
             self.assertEqual(pyfunc(*args), cfunc(*args),
                              msg='failed on {}'.format(args))
 
+    def test_islower(self):
+        pyfunc = islower_usecase
+        cfunc = njit(pyfunc)
+        lowers = [x.lower() for x in UNICODE_EXAMPLES]
+        extras = ["AA12A", "aa12a", "大AA12A", "大aa12a", "AAAǄA", "A 1 1 大"]
+
+        # Samples taken from CPython testing:
+        # https://github.com/python/cpython/blob/201c8f79450628241574fba940e08107178dc3a5/Lib/test/test_unicode.py#L586-L600
+        cpython = ['\u2167', '\u2177', '\U00010401', '\U00010427', '\U00010429',
+                   '\U0001044E', '\U0001F40D', '\U0001F46F']
+        fourxcpy = [x * 4 for x in cpython]
+
+        for s in UNICODE_EXAMPLES + lowers + [""] + extras + cpython + fourxcpy:
+            self.assertEqual(pyfunc(s), cfunc(s),
+                             msg='Results of interpreted and compiled "{}".islower() should be equal'.format(s))
+
+    def test_lower(self):
+        pyfunc = lower_usecase
+        cfunc = njit(pyfunc)
+        extras = ["AA12A", "aa12a", "大AA12A", "大aa12a", "AAAǄA", "A 1 1 大"]
+
+        # Samples taken from CPython testing:
+        # https://github.com/python/cpython/blob/201c8f79450628241574fba940e08107178dc3a5/Lib/test/test_unicode.py#L748-L758
+        cpython = ['\U00010401', '\U00010427', '\U0001044E', '\U0001F46F', '\U00010427\U00010427',
+                   '\U00010427\U0001044F', 'X\U00010427x\U0001044F', '\u0130']
+
+        # special cases for sigma from CPython testing:
+        # https://github.com/python/cpython/blob/201c8f79450628241574fba940e08107178dc3a5/Lib/test/test_unicode.py#L759-L768
+        sigma = ['\u03a3', '\u0345\u03a3', 'A\u0345\u03a3', 'A\u0345\u03a3a', '\u03a3\u0345 ', '\U0008fffe', '\u2177']
```

Comment:
These lines are too long. <80 chars please.

Reply:
Ok, I will change it.